handlers/tokeninfo/jwt/tokeninfo.go

```diff
@@ -148,7 +168,7 @@ func claimAsString(t *jwt.Token, claim string) (string, bool) {
 	return "", false
 }
 
-func claimAsInt64(t *jwt.Token, claim string) (int64, bool) {
+func ClaimAsInt64(t *jwt.Token, claim string) (int64, bool) {
```

Comment:
You don't need to export ClaimAsInt64() to other packages, so use lowercase claimAsInt64() instead.